blocks/header/header.css

```diff
@@ -0,0 +1,57 @@
+body {
+    max-width: 1673px;
+    margin: 0 auto;
+
+    background-color: white;
+}
+.header {
+    z-index: 1;
+
+    width: 100%;
+    height: 648px;
+
+    background-image: url(../../assets/img/header/sochi-2258855.png);
+}
+.header__logo {
+    margin-top: 21px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.header__title{
+margin-top:2px;
+}
+
+.header__subtitle {
+    font-family: 'Roboto Slab','Arial', sans-serif;
+    font-size: 100px;
+    font-weight: 400;
+
+    margin-top: -13px;
+    margin-bottom:0;
+
+    text-align: center;
+    letter-spacing: -3px;
+    text-transform:uppercase;
+
+    color: #0d3351;
+}
+.header__button {
+    margin-top: 33px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+.decor-line {
+    z-index: 2;
+
+    margin-right: auto;
+    margin-left:auto;
+    position: absolute;
+
+    left: 0;
+    right: 10px;
+   
+    margin-top: -179px;

```

Comment:
I guess we can use `top/bottom` to control this as we already have this element as `absolute`.